### ISSUE TITLE
Account screen Freezes with blocking layer (persistent after restart) upon "Create Account"

### ISSUE THREAD

**Issue by fraenk** [aaroncox/vessel#71]
When clicking "Create Account" the account screen immediately gets blocked.

This blocking persists after a relaunch of vessel. So far I've only been able to resolve the issue by manually deleting vessel's account data and reimporting the wallet from scratch.

Here's the Errors shown in the console:

```
bundle.js:1 Uncaught TypeError: Cannot read property 'amount' of undefined
    at Le.render (bundle.js:1)
    at he._renderValidatedComponentWithoutOwnerOrContext (bundle.js:346)
    at he._renderValidatedComponent (bundle.js:346)
    at he.performInitialMount (bundle.js:346)
    at he.mountComponent (bundle.js:346)
    at Object.mountComponent (bundle.js:346)
    at ve.mountChildren (bundle.js:346)
    at ve._createInitialChildren (bundle.js:346)
    at ve.mountComponent (bundle.js:346)
    at Object.mountComponent (bundle.js:346)
    at he.performInitialMount (bundle.js:346)
    at he.mountComponent (bundle.js:346)
    at Object.mountComponent (bundle.js:346)
    at ve.mountChildren (bundle.js:346)
    at ve._createInitialChildren (bundle.js:346)
    at ve.mountComponent (bundle.js:346)
events.js:163 Uncaught Error: getaddrinfo ENOTFOUND steemd.steemit.com steemd.steemit.com:443
    at errnoException (dns.js:28:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:73:26)
```

and a screenshot:
![image](https://user-images.githubusercontent.com/10187569/34825911-be145d34-f6d4-11e7-8754-09888941dbed.png)

i've tried some different nodes but the problem seems to occur everywhere?!

**Comment by fraenk** [comment]
I had hoped this would still be related to the node problems, but the problem remains with 0.2.5.

I have also manually set the buildteam rpc node after installation just in case... 

Create.js Line 360:
`<List.Item>STEEM: {delegation.fee.amount}</List.Item>`
Cannot read property 'amount' of undefined

**Comment by aaroncox** [comment]
Can you provide the full errors in the console now with 0.2.5? 

The errors originally provided in the screenshot above show it's a connection a connection issue (trying to use steemd.steemit.com), but that doesn't seem to be happening anymore.

**Comment by fraenk** [comment]
Hi @aaroncox,

here's the full error reported loggen in the console after a fresh relaunch of vessel and then activating the "accounts" tab:

`bundle.js:359  action ACCOUNT_DATA_UPDATE_PENDING @ 19:39:01.901
3bundle.js:1 Unhandled rejection Error: invalid url
    at be.te (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:1162539)
    at new be (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:1166485)
    at file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:900704
    at ae.$._execute (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:459140)
    at ae._resolveFromExecutor (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:495080)
    at new ae (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:486494)
    at je.value (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:900667)
    at je.value (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:901251)
    at Se.value (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:884179)
    at Se.Ee.(anonymous function) [as getDynamicGlobalPropertiesWith] (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:881966)
    at Se.Ee.(anonymous function) [as getDynamicGlobalProperties] (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:882243)
    at file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:17518
    at file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:558972
    at Object.refreshGlobalProps (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:559834)
    at Qe.changeNode (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:212383)
    at Qe.componentWillMount (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:212490)
Ke @ bundle.js:1
redux-logger.js:1  action ACCOUNT_DATA_UPDATE_PENDING @ 19:39:11.903
2debuggability.js:868 Unhandled rejection Error: invalid url
    at be.te (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:1162539)
    at new be (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:1166485)
    at file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:900704
    at ae.$._execute (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:459140)
    at ae._resolveFromExecutor (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:495080)
    at new ae (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:486494)
    at je.value (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:900667)
    at je.value (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:901251)
    at Se.value (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:884179)
    at Se.Ee.(anonymous function) [as getDynamicGlobalPropertiesWith] (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:881966)
    at Se.Ee.(anonymous function) [as getDynamicGlobalProperties] (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:882243)
    at file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:17518
    at file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:558972
    at Object.refreshGlobalProps (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:359:559834)
    at Qe.changeNode (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:212383)
    at Qe.componentWillMount (file:///C:/Users/fraenk/AppData/Local/Programs/vessel/resources/app.asar/dist/bundle.js:1:212490)
Ke @ debuggability.js:868
Se @ debuggability.js:593
Ce @ debuggability.js:618
$._notifyUnhandledRejection @ debuggability.js:64
(anonymous) @ debuggability.js:44
redux-logger.js:1  action @@router/LOCATION_CHANGE @ 19:39:15.094
Create.js:360 Uncaught TypeError: Cannot read property 'amount' of undefined
    at De.render (Create.js:360)
    at he._renderValidatedComponentWithoutOwnerOrContext (ReactCompositeComponent.js:808)
    at he._renderValidatedComponent (ReactCompositeComponent.js:820)
    at he.performInitialMount (ReactCompositeComponent.js:362)
    at he.mountComponent (ReactCompositeComponent.js:255)
    at Object.mountComponent (ReactReconciler.js:43)
    at ve.mountChildren (ReactMultiChild.js:230)
    at ve._createInitialChildren (ReactDOMComponent.js:701)
    at ve.mountComponent (ReactDOMComponent.js:520)
    at Object.mountComponent (ReactReconciler.js:43)
    at he.performInitialMount (ReactCompositeComponent.js:368)
    at he.mountComponent (ReactCompositeComponent.js:255)
    at Object.mountComponent (ReactReconciler.js:43)
    at ve.mountChildren (ReactMultiChild.js:230)
    at ve._createInitialChildren (ReactDOMComponent.js:701)
    at ve.mountComponent (ReactDOMComponent.js:520)
render @ Create.js:360
_renderValidatedComponentWithoutOwnerOrContext @ ReactCompositeComponent.js:808
_renderValidatedComponent @ ReactCompositeComponent.js:820
performInitialMount @ ReactCompositeComponent.js:362
mountComponent @ ReactCompositeComponent.js:255
mountComponent @ ReactReconciler.js:43
mountChildren @ ReactMultiChild.js:230
_createInitialChildren @ ReactDOMComponent.js:701
mountComponent @ ReactDOMComponent.js:520
mountComponent @ ReactReconciler.js:43
performInitialMount @ ReactCompositeComponent.js:368
mountComponent @ ReactCompositeComponent.js:255
mountComponent @ ReactReconciler.js:43
mountChildren @ ReactMultiChild.js:230
_createInitialChildren @ ReactDOMComponent.js:701
mountComponent @ ReactDOMComponent.js:520
mountComponent @ ReactReconciler.js:43
performInitialMount @ ReactCompositeComponent.js:368
mountComponent @ ReactCompositeComponent.js:255
mountComponent @ ReactReconciler.js:43
oe @ ReactMount.js:102
perform @ Transaction.js:141
se @ ReactMount.js:124
batchedUpdates @ ReactDefaultBatchingStrategy.js:58
batchedUpdates @ ReactUpdates.js:95
_renderNewRootComponent @ ReactMount.js:317
_renderSubtreeIntoContainer @ ReactMount.js:398
renderSubtreeIntoContainer @ ReactMount.js:340
value @ Portal.js:319
value @ Portal.js:254
ne.notifyAll @ CallbackQueue.js:74
close @ ReactReconcileTransaction.js:79
closeAll @ Transaction.js:207
perform @ Transaction.js:154
perform @ Transaction.js:141
perform @ ReactUpdates.js:87
Se @ ReactUpdates.js:170
closeAll @ Transaction.js:207
perform @ Transaction.js:154
batchedUpdates @ ReactDefaultBatchingStrategy.js:60
batchedUpdates @ ReactUpdates.js:95
dispatchEvent @ ReactEventListener.js:145
events.js:163 Uncaught Error: getaddrinfo ENOTFOUND steemd.steemit.com steemd.steemit.com:443
    at errnoException (dns.js:28)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:73)`

**Comment by aaroncox** [comment]
What's stopping you is actually `invalid url`, so it sounds like the URL in the preferred steem node isn't correctly entered.

**Comment by fraenk** [comment]
that's rather weird... the node is the rpc.buildteam.io and everything else works.

The wallet is usable upon launch and wallet data is accurate and up-to-date. Only once I enter the account screen does this error appear.

I'll try again to clear the local storage and reimport the wallet, let's see how that goes down.

**Comment by fraenk** [comment]
Ok... managed to get the wallet rolling again... i haven't yet clicked "create account" but the url error pops up right ahead.

![image](https://user-images.githubusercontent.com/10187569/34894704-a42ec7f6-f7e2-11e7-8163-0aaad6e76d58.png)

i double checked, the node is still set to buildteam

**Comment by aaroncox** [comment]
Oh, dammit. I know exactly what this is now. There's a 2nd library that's used to estimate the account creation fee and the defaults on that are probably hitting steemd.steemit.com. 

I'll get a fix implemented for the next version (0.2.6). Sorry about the confusion and thanks for going back and forth with me until I figured it out :)

**Comment by fraenk** [comment]
fixed as promised in 0.2.6

gotta love you for the work you do @aaroncox 

steem-on
